build/build.sh

```diff
@@ -325,11 +367,18 @@ function Build {
     if [ $hostType = "core" ]
     then
       msbuildToUse=$(QQ "$bootstrapRoot/netcoreapp2.1/MSBuild/MSBuild.dll")
+    elif [ "$hostType" = "mono" ]
+    then
+      msbuildToUse="$bootstrapRoot/net461/MSBuild/15.0/Bin/MSBuild.dll"
+      msbuildHost="mono"
+
+      properties="$properties /p:MSBuildExtensionsPath=$bootstrapRoot/net461/MSBuild/"
     else
       ErrorHostType
     fi
 
-    export ArtifactsDir="$ArtifactsDir/2"
+    # forcing the slashes here or they get normalized and lost later
+    export ArtifactsDir="$ArtifactsDir\\2\\"
```

Comment:
why backslashes?

Reply:
The `2Debug` directory names seemed like a mistake. This change would make the directories `artifacts/2/Debug-Mono/...`. But if that is intentional, then we can skip this change.